src/app/components/list/view-switcher/index.html

```diff
@@ -21,44 +21,44 @@
       <h3 class="sky-text-success">Use when</h3>
         <sky-docs-visual-example>
           <sky-docs-visual-example-text>
-              <p>Use the list view switcher when the user needs to switch between different views of the same data.</p>
+              <p>Use list view switchers when users need to switch between different views of the same data.</p>
           </sky-docs-visual-example-text>
           <sky-docs-visual-example-image>
             <div>
               <img src="~/assets/img/guidelines/list-view-switcher/use-when.png" />
-              <div class="caption caption-success sky-text-success">Do use the list view switcher when there is more than one view of the data.</div>
+              <div class="caption caption-success sky-text-success">Do use list view switchers when more than one view of data is available to users.</div>
             </div>
           </sky-docs-visual-example-image>
         </sky-docs-visual-example>
-      <h3 class="sky-text-success">Don't use when</h3>
+      <h3 class="sky-text-danger">Don't use when</h3>
         <sky-docs-visual-example>
           <sky-docs-visual-example-text>
-              <p>Don't use the list view switcher when there is only one view of the data.</p>
+              <p>Don't use list view switchers when only one view of data is available.</p>
           </sky-docs-visual-example-text>
           <sky-docs-visual-example-image>
             <div>
               <img src="~/assets/img/guidelines/list-view-switcher/dont-use-when.png" />
-              <div class="caption caption-danger sky-text-danger">Don't use the list view switcher when there is only one view of the data.</div>
+              <div class="caption caption-danger sky-text-danger">Don't use list view switchers when users cannot switch between different views.</div>
             </div>
           </sky-docs-visual-example-image>
         </sky-docs-visual-example>
     </sky-demo-page-content>
 
     <sky-demo-page-content sectionHeading="Anatomy">
       <sky-docs-anatomy imagePath="~/assets/img/guidelines/list-view-switcher/anatomy.png">
-        <sky-docs-anatomy-item anatomyNumber="1">Active view</sky-docs-anatomy-item>
-        <sky-docs-anatomy-item anatomyNumber="2">Inactive view</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="1">Active view button</sky-docs-anatomy-item>
```

Comment:
@Blackbaud-ToddRoberts, JFYI that I added “button” to the item names for consistency with our naming for similar items in the datepicker, file attachment, and phone field Anatomy docs. Just calling this out in case you see any problem with this change.

Reply:
good call